core/src/main/java/io/grpc/Context.java

```diff
@@ -510,38 +526,38 @@ public C call() throws Exception {
 
   /**
    * Wrap an {@link Executor} so that it executes with this context as the {@link #current} context.
-   * It is generally expected that {@link #propagate(Executor)} would be used more commonly than
-   * this method.
+   * It is generally expected that {@link #currentContextExecutor(Executor)} would be used more
```

Comment:
Late comment, but if fixed context is edge case vs current, maybe have an example of where it might be useful?